### PR TITLE
feat: scientificNotationToDecimal + tests

### PR DESCRIPTION
**Pull request by farzad-k2003**
feature of converting science notation numbers to normal strings was added and tested

### CODE REVIEW

**Comment by hosseinmd** [comment]
LGTM

File: src/core/scientificNotation.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function scientificNotationToDecimal(value: number | string): string {
-  if (!value) return "";
+  if (value == null) return "";
```
Comment (hosseinmd) [inline src/core/scientificNotation.ts:2]: Null is not type of value in StarkString internal value (_value) is always string

**Review by hosseinmd — CHANGES_REQUESTED** [review]

**Comment by hosseinmd** [comment]
Thank you for your contributing.

**Comment by fardad-dev** [comment]
Released v0.4.4

**Comment by farzad-k2003** [comment]
my pleasure

**Pull request merged**